### DISCUSSION TITLE
Kernel Panic on installing

### DISCUSSION THREAD

**Discussion by killmasta93** [Security-Onion-Solutions/securityonion#12266] [2.4] [answered]
Hi
Currently trying to install security onion on proxmox 7.4-3 but as soon as it starts to install i get this error
![image](https://github.com/Security-Onion-Solutions/securityonion/assets/13953629/1c65307a-7363-474f-a613-109793a548a6)
using the latest version of security onion,
A while back i installed perfectly with version  2.3.250-20230519

Thank you

**Comment by reyesj2** [comment, marked as answer]
Installing on proxmox you need to set the CPU type to 'host'
https://docs.securityonion.net/en/2.4/proxmox.html#cpu

**Comment by Sasquatch8946** [comment]
I'm currently running into the exact same issue, and I had already set the CPU type to 'host.' My problem could be old hardware. I'm trying to get this running on a Dell Poweredge 2900.

**Reply by killmasta93** [reply]
put max on CPU 
also got it working like that 
![image](https://github.com/Security-Onion-Solutions/securityonion/assets/13953629/364ff5ff-c92b-492c-90d7-10692f615716)
